File: A_Star.cpp
#include <iostream>
using namespace std;

int main(){
    int x;
    cin >> x;
    int ans = 0;
    if (x % 100 == 0){
        ans = 100;
    }else{
        int y = (x / 100 + 1) * 100;
        ans = y - x;
    }
    cout << ans << endl;
}

File: A-Star.go
package main
import (
	"fmt"
)

func main(){
	var x, ans int
	fmt.Scan(&x)
	ans = 1
	for i := x + 1; i <= 1000000; i ++ {
		if i % 100 == 0 {
			break
		} else {
			ans += 1
		}
	}

	fmt.Println(ans)

}